core/src/main/java/org/eclipse/hono/util/RegistryManagementConstants.java

```diff
@@ -124,6 +124,17 @@
      */
     public static final String PARAM_SORT_JSON = "sortJson";
 
+    /**
+     * The name of the field that contains the total number of objects in the result set of the search devices
+     * operation.
+     */
+    public static final String FIELD_SEARCH_DEVICES_RESULT_TOTAL = "total";
+
+    /**
+     * The name of the field that contains the result of the search devices operation.
+     */
+    public static final String FIELD_SEARCH_DEVICES_RESULT = "result";
+
```

Comment:
I am pretty sure that we will re-use these constants for searching tenants and/or credentials. How about naming them more generically, e.g. *FIELD_RESULT_SET_SIZE* and *FIELD_RESULT_SET_PAGE*?

Reply:
I have renamed them as _FIELD_RESULT_SET_SIZE_ and _FIELD_RESULT_SET_PAGE_.